### PR TITLE
improve coverage

### PR DESCRIPTION
**Pull request by Khabib73**
improve coverage in [calculators.py](https://github.com/yandex/ch-backup/blob/main/ch_backup/calculators.py) to 100%

## Summary by Sourcery

Increase test coverage for calculator alignment utilities

Tests:
- Add parameterized tests for calc_aligned_data_size covering various alignments and empty input
- Add integration-style tests for calc_aligned_files_size_scan including exclusion of specific file names

### CODE REVIEW

File: tests/unit/test_calculators.py [hunk 1]
```diff
@@ -16,6 +18,45 @@
...
+        ([b"a", b"b"], 1, 2),
+        ([b"a"], 512, 512),
+        ([b"a" * 512], 512, 512),
+        ([b"a" * 513], 512, 1024),
+        ([b"a", b"b" * 512], 512, 1024),
+        ([b"a", b"b"], 2, 4),
+        ([b"ab", b"cd"], 2, 4),
+        ([], 512, 0),
+    ],
+)
```
[the remark this answers is not in the document]
Comment (Khabib73) [reply tests/unit/test_calculators.py:34]: done

**Comment by Khabib73** [comment]
@aalexfvk, Hi! PR is ready for review :)

**Review by aalexfvk — APPROVED** [review]

**Comment by Khabib73** [comment]
Thanks!)
